validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -307,8 +308,16 @@ public void sendSignedBlock(final SignedBeaconBlock block) {
     eventBus.post(new ProposedBlockEvent(block));
   }
 
-  private boolean isSyncActive() {
-    return !syncStateTracker.getCurrentSyncState().isInSync();
+  @VisibleForTesting
+  boolean isSyncActive() {
+    return syncStateTracker.getCurrentSyncState().isStartingUp()
+        || (syncStateTracker.getCurrentSyncState().isSyncing() && headBlockIsTooFarBehind());
```

Comment:
I'd extract `getCurrentSyncState` to a variable so we have a consistent view of it rather than it potentially changing part way through this logic (should still work out fine but is easier to reason about if its fixed).